ui/components/Navigator.js

```diff
@@ -235,15 +240,16 @@ const categories = [
   },
   {
     id: "Performance",
-    icon: <FontAwesomeIcon icon={faTachometerAlt} transform="shrink-2" fixedWidth />,
+    icon:
+      <FontAwesomeIcon icon={faTachometerAlt} transform="shrink-2" style={{ height: "1.21rem", width: "1.21rem", fontSize: "1.21rem" }} />,
```

Comment:
@Ayush-Khandelwal-007 can't the `drawerIcons` class be used here instead of adding style to each of the icons?

Reply:
okay , I'll just check it and update to the cleanest way possible.

Reply:
I can't use class `drawerIcons ` for the elements defined outside the class. So I just cleaned it up a bit.
and can you please help me resolve the issue in the below checks?